Cargo.toml

```diff
@@ -40,10 +40,10 @@ quote = "1.0"
 regex = "1.3"
 regex-syntax = "0.8"
 serde = "1.0"
-sparsevec = "0.2"
+sparsevec = { git = "https://github.com/softdevteam/sparsevec.git", rev = "dcf739917357c4a5bd3c358e48de9613360b40fe" }
```

Comment:
We'll want to fix this before committing.

Reply:
Should be fixed in [51e7f08](https://github.com/softdevteam/grmtools/pull/530/commits/51e7f08cb0b055b580a597388e273ab1af2ce1a5)